consumer/src/main/kotlin/streams/StreamsTopicService.kt

```diff
@@ -0,0 +1,38 @@
+package streams
+
+import org.neo4j.kernel.impl.core.EmbeddedProxySPI
+import org.neo4j.kernel.impl.core.GraphPropertiesProxy
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import java.lang.IllegalArgumentException
+
+
+private const val STREAMS_TOPIC_KEY: String = "streams.sink.topic."
+
+class StreamsTopicService(private val db: GraphDatabaseAPI, private val streamsSinkConfiguration: StreamsSinkConfiguration) {
+    private val properties: GraphPropertiesProxy = db.dependencyResolver.resolveDependency(EmbeddedProxySPI::class.java).newGraphPropertiesProxy()
+
+    fun set(topic: String, query: String) {
```

Comment:
do we expose these 3 operations via procedures?

Reply:
perahps create an issue

Reply:
> perahps create an issue

Is the #46, is based on what we have talked about on Slack. Give it a check if you want update it